sympy/physics/control/lti.py

```diff
@@ -4263,3 +4263,119 @@ def is_controllable(self):
 
         """
         return self.controllability_matrix().rank() == self.num_states
+
+class PIDController(TransferFunction):
+    """
+    A PID Controller class that uses Parallel connection to combine the P, I, and D components.
+
+    Parameters
+    ==========
+    KP : Expr, Number
+        Proportional gain.
+    KI : Expr, Number
+        Integral gain.
+    KD : Expr, Number
+        Derivative gain.
+    TF : Expr, Number
+        Derivative filter time constant
+    s : Symbol
+        The complex frequency variable.
+
+    Examples
+    ========
+    >>> from sympy.abc import s
+    >>> from sympy.physics.control.lti import PIDController
+    >>> KP, KI, KD, TF = 1, 0.1, 0.01, 0
+    >>> pid = PIDController(KP, KI, KD, TF, s)
+    >>> print(pid)
+    PIDController(1, 0.1, 0.01, 0, s)
+    >>> pid.doit() #Converts PIDController into TransferFunction
+    TransferFunction(s*(0.01*s + 1) + 0.1, s, s)
+    >>> pid.var
+    s
+    >>> pid.KP
+    1
+    """
+
+    def __new__(cls, KP = None, KI = None, KD = None, TF = None, var = None):
```

Comment:
The capital letters and spacing around equals does not follow PEP8.

Reply:
Shouldn't var be a required argument?

Reply:
Yes, it doesn't need a default value and a check if it has been given.

Reply:
I think you did that because you cannot have a positional argument without a default value after a positional argument with a default value in a function in Python, if I'm correct ?

Reply:
Yes, It was showing error otherwise.